### ISSUE TITLE
FA102 raised on python 3.10

### ISSUE THREAD

**Issue by DavideCanton** [TylerYep/flake8-future-annotations#15]
Hi, thanks for the plugin!

I have the following configuration:

```py
# file foo.py

def bar(foo: list[int]) -> None:
    print(foo)
```

```
# file setup.cfg
[flake8]
extend-ignore = D
extend-select = FA
check-future-annotations = true
```

```
$ python -V
Python 3.10.12
$ pip list
Package                   Version
------------------------- -------
flake8                    7.0.0  
flake8-future-annotations 1.1.0  
mccabe                    0.7.0  
pip                       23.0.1 
pycodestyle               2.11.1 
pyflakes                  3.2.0  
setuptools                65.5.0 
```

With this setup, running `flake8 foo.py` raises a FA102 warning.

I read in the readme that
> If the --check-future-annotations option is set, missing from __future__ import annotations will be reported because the following code will error on Python versions older than 3.10 (this check does not output anything for versions 3.10+)

What I thought is that with a python version >= 3.10 no warning should be raised if using simplified type annotations, am I wrong?

**Comment by TylerYep** [comment]
If you are on Python 3.10+, then you shouldn't set `check-future-annotations = true`.

**Comment by DavideCanton** [comment]
This is not what the readme states, though. I thought that the check had version detection in the code, maybe this could be added to the readme?

**Comment by TylerYep** [comment]
Oh, I see now. I can update the readme to clarify that the setting doesn't need to be enabled on Python 3.10.

Version detection can be error-prone, so I likely won't add it to this plugin, especially since this equivalent check now exists on newer tools like Ruff.

**Comment by DavideCanton** [comment]
I think it should be enough :) thanks!

**Comment by TylerYep** [comment]
Fixed, thanks!
